tests/integration/server/api/users/[id]/index.put.test.ts

```diff
@@ -0,0 +1,37 @@
+import { describe, expect, it } from "vitest"
+import { setup, $fetch } from "@nuxt/test-utils/e2e"
+import { v4 as uuidv4 } from "uuid"
+import prisma from "../../../../../../server/utils/prisma"
+
+describe("Api Route /api/users", async () => {
+  await setup()
+
+  it ("Should update a user", async () => {
```

Comment:
Same here, name it should update a user by username and add another test which checks it also works with the id